### PR TITLE
Add documentation on SetupRepository.bat script

### PR DESCRIPTION
**Pull request by chrisfromwork**
This review contains a few changes:

1) The landing page now has information on the SetupRepository.bat step
2) SetupRepository setup scripts are updated to configure the repo for crlf line endings. This should prevent line ending changes that make viewing pull requests more difficult.
3) A few documentation pages calling out a deleted calibration doc have been fixed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,31 @@ Spectator View is an augmented reality product that enables viewing HoloLens exp
...
-### Obtaining the Code
+### Obtaining the code
 
 Active development occurs in MixedReality-SpectatorView's master branch. It's suggested to consume Spectator View code from one of the release branches compared to the master branch.
 
 >Note: Spectator view uses symbolic linked directories in its sample projects, which results in large file paths. It's suggested to place your Unity project in a short named directory (such as C:\sv). Otherwise, file paths may become too long to resolve in Unity.
 
+### Setting up your local environment
+
+The MixedReality-SpectatorView repository uses unity packages, git submodules and symbolic linked directories for obtaining and referencing external dependencies. Prior to opening any Unity projects, its suggested to run `tools/Scripts/SetupRepository.bat` as an administrator on your PC (On Mac or Linux, you can run `/tools/scripts/SetupRepository.sh`). This script has a few purposes:
```
Comment (matthejo) [inline README.md]: >unity [](start = 47, length = 5)

Nit: capitalize Unity

File: tools/Scripts/SetupRepository.ps1 [hunk 2]
```diff
@@ -37,15 +37,23 @@ Set-Location (Split-Path $MyInvocation.MyCommand.Path)
 # for inter-directory linking to build the HolographicCamera.Unity project
 # as well as for the samples to include submodule repository content
 # via directory symlinks.
+Write-Output "Enabling symbolic link directories for the repository."
```
Comment (matthejo) [inline tools/Scripts/SetupRepository.ps1]: >directories [](start = 37, length = 11)

Technically this enables any kind of symbolic link, not just for directories

File: tools/Scripts/SetupRepository.ps1 [hunk 3]
```diff
@@ -37,15 +37,23 @@ Set-Location (Split-Path $MyInvocation.MyCommand.Path)
...
 git submodule update --init
 
 # If any links were created to the submodules but were broken,
 # restore those symlinks now that the submodules are cloned.
+Write-Output "Updating all symbolic link directories."
 dir "$PSScriptRoot\..\..\" -Recurse -File | ?{$_.LinkType -eq "SymbolicLink" } | Restore-SymbolicLink
 
 # If any links were created before the repository was configured to use
 # symlinks, those links need to be restored.
+Write-Output "Fixing any custom symbolic links."
```
Comment (matthejo) [inline tools/Scripts/SetupRepository.ps1]: >Fixing any custom symbolic links. [](start = 14, length = 33)

I would just lump this second command in with the first as far as reporting status. If you want to describe them separately, I would call the first one "Updating symbolic links into submodules" and the second "Fixing any symbolic links created before symbolic linking was enabled for the repository"

**Comment by chrisfromwork** [comment]
I'm going to go ahead and merge this review. https://github.com/microsoft/MixedReality-SpectatorView/pull/130 will conflict with these changes and needs information on the SetupRepository.bat script.

**Pull request merged**